djangocms_navigation/templatetags/navigation_menu_tags.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from django import template
+
+from cms.models import Page
+from menus.menu_pool import menu_pool
+
+from classytags.arguments import Argument
+from classytags.core import Options
+from classytags.helpers import InclusionTag
+
+
+register = template.Library()
+
+
+class Navigation_ShowBreadcrumb(InclusionTag):
```

Comment:
Violates PEP 008 naming where a class name should be upper cased words. Needs to be changed to: NavigationBreadcrumb